bundle-workflow/src/manifests/test_manifest.py

```diff
@@ -15,6 +15,7 @@ class TestManifest(Manifest):
         schema-version: '1.0'
         components:
           - name: index-management
+            working-directory: optional relative directory to run commands in
```

Comment:
can this be an optional parameter, so only outlier plugins have this in their configuration?

Reply:
It is an optional plugin, only the outlier plugins have this specified, for example: https://github.com/opensearch-project/opensearch-build/pull/529/files#diff-7e054842e270f704c5ec5d3728ffce2dda6533adb3a626822b1ab586aa79acc8R55

Reply:
maybe rename this config param to `gradle-path` if that's what this represents.

Reply:
`gradle-path` would be specific to gradle plugins and we won't be able to re-use it for dashboards plugin. WDYT about `testing-directory`?

Reply:
We already have `working-directory` for build, so naming-wise that matches and I personally like it.

Reply:
Thats true, `testing-directory` would not represent it completely. I kept it `working-directory` to keep it consistent with the build side.

Reply:
Sounds good, didn't know we use similar name for build. Makes sense to keep it consistent. Thanks!

Reply:
It was added as part of https://github.com/opensearch-project/opensearch-build/pull/402.

Reply:
If you are reading the input manifest anywhere here, you could just take it out of there.

Reply:
I don't think we are reading the input manifest here, its just the bundle, build and test manifest.